examples/mnist-cnn.py

```diff
@@ -56,22 +53,25 @@ def __init__(self, x_test):
 
     def on_train_begin(self, logs=None):
         st.header("Summary")
-        self._summary_chart = self._create_chart("area", 300)
+        empty_chart = pd.DataFrame({"loss": [], "acc": []})
+        self._summary_chart = st.area_chart(empty_chart)
         self._summary_stats = st.text("%8s :  0" % "epoch")
         st.header("Training Log")
 
     def on_epoch_begin(self, epoch, logs=None):
         self._ts = time.time()
         self._epoch = epoch
         st.subheader("Epoch %s" % epoch)
-        self._epoch_chart = self._create_chart("line")
+        empty_chart = pd.DataFrame({"loss": [], "acc": []})
+        self._epoch_chart = st.line_chart(empty_chart)
```

Comment:
Actually, I forgot about a requirement we have for this conversion to VegaLite (sorry!!!!).

We're like to be able to do this:
```
chart = st.line_chart()  # ← No arguments!
chart.add_rows(df)
```

This is supported today, but this PR breaks it. Any idea how hard it would be to support this again?

Reply:
There is a cypress test in which line_chart, bar_chart, and area_chart should throw an exception if there is no data.

Reply:
Done in commit https://github.com/streamlit/streamlit/pull/56/commits/0877e6ca1842253a74422aef3f9243797d21e7cd